frontstage/controllers/collection_instrument_controller.py

```diff
@@ -86,3 +86,14 @@ def upload_collection_instrument(upload_file, case_id, party_id):
         raise ApiError(logger, response)
 
     logger.info('Successfully uploaded collection instrument', case_id=case_id, party_id=party_id)
+
+
+def check_collection_instrument_size(upload_file, party_id, case_id):
+    logger.info('Checking the file size', party_id=party_id, case_id=case_id)
+    upload_file_readable = upload_file.read()
+    upload_file_size = len(upload_file_readable)
```

Comment:
```suggestion
    upload_file_size = len(upload_file.read())
```

You only use the variable once, so might as well flatten it